Ignore optional installs fails to find scrypt.js

<!--- Provide a general summary of the issue in the Title above -->
Adding ganache-core as a dependency and installing all dependencies with:
```
yarn --frozen-lockfile --ignore-optional
```
Results in Ganache failing to load as it cannot find the scrypt dependency.
```
ganache-core-optional λ node
> const ganache = require("ganache-core");
{ Error: Cannot find module 'scrypt.js'
    at call (/Users/jacob/tmp/zz/ganache-core-optional/node_modules/ganache-core/build/webpack:/Ganache/webpack/bootstrap:19:1)
    at Object.__webpack_require__ (/Users/jacob/tmp/zz/ganache-core-optional/node_modules/ganache-core/build/webpack:/Ganache/node_modules/ethereumjs-wallet/index.js:8:16)
    at call (/Users/jacob/tmp/zz/ganache-core-optional/node_modules/ganache-core/build/webpack:/Ganache/webpack/bootstrap:19:1)
    at Object.require (/Users/jacob/tmp/zz/ganache-core-optional/node_modules/ganache-core/build/webpack:/Ganache/external "scrypt.js":1:1)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15) code: 'MODULE_NOT_FOUND' }
```

<!--- DO NOT PASTE ONLY IMAGES OF TERMINAL OUTPUT AS AN ISSUE --->

<!--- WINDOWS USERS: Please READ all of the error / debug output and make sure that the issue 
      isn't due to you missing something before creating an issue. A large portion of Windows 
      issues turn out to be easily self-fixable if you actually read the output before dumping
      it in an issue here. Thanks! ---> 

## Expected Behavior
Ganache-core doesn't fail on load if optional dependencies are skipped.
<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->

## Current Behavior
Ganache-core fails on load if optional dependencies are skipped.
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Add ganache-core as a dependency
2. Install packages ignoring optionals: `yarn --frozen-lockfile --ignore-optional`
3. Load ganache-core
4. Cannot find module `scrypt.js`

## Context
<!--- How has this issue affected you? What are you trying to accomplish? -->
We are publishing a lean and mean Docker image and wanting to remove sizeable dependencies which are not required.
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: 2.2.0
* Environment name and version (e.g. PHP 5.4 on nginx 1.9.1):
* Server type and version:
* Operating System and version: Linux
* Link to your project:

--- a/package.json
+++ b/package.json
@@ -64,6 +64,7 @@
     "ethereumjs-wallet": "0.6.3",
     "ethers": "4.0.26",
     "husky": "^1.1.3",
+    "js-scrypt": "0.2.0",
     "lint-staged": "^8.0.4",
     "memdown": "^1.3.1",
     "mocha": "5.2.0",

--- a/webpack/base.webpack.config.js
+++ b/webpack/base.webpack.config.js
@@ -15,7 +15,7 @@ module.exports = (override) => {
         // webpack these modules:
         // we actually only care about scrypt and eth-block-tracker here, as those are the only native modules
         // but webpack won't detect them if we don't traverse the dependency tree to get to them
-        if (/^(ethereumjs-wallet|scrypt|web3|eth-block-tracker)(\/.*)?$/.test(request)) {
+        if (/^(ethereumjs-wallet|scrypt|web3|web3-eth|web3-eth-accounts|eth-block-tracker)(\/.*)?$/.test(request)) {
           return callback();
         }
 

